### PR TITLE
Sprint1

### PR DESCRIPTION
**Pull request by abdullah2021skipq**
Abdullah Zaman Babar

### CODE REVIEW

File: AbdullahZaman/sprint1/PCRepoAZB/resources/s3bucket.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import boto3
+
```
Comment (Ayesha-zakria) [inline AbdullahZaman/sprint1/PCRepoAZB/resources/s3bucket.py:2]: your code is well presented

Comment (abdullah2021skipq) [reply AbdullahZaman/sprint1/PCRepoAZB/resources/s3bucket.py:2]: Thank You

File: AbdullahZaman/sprint1/PCRepoAZB/resources/webhealth_lambda.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	avail = get_availability()
+	dimensions = [
+		{"Name": "URL", "Value": constants.URL_TO_MONITOR}
+		]
+	
+	cw.put_data(constants.URL_MONITOR_NAMESPACE,constants.URL_MONITOR_NAME_Availability, dimensions, avail)
+	
+	latency = get_latency()
+	dimensions = [
+		{"Name": "URL", "Value": constants.URL_TO_MONITOR}
```
Comment (Ayesha-zakria) [inline AbdullahZaman/sprint1/PCRepoAZB/resources/webhealth_lambda.py:19]: you may write here in comments how cdk/ boto3 takes dimensions .

Comment (abdullah2021skipq) [reply AbdullahZaman/sprint1/PCRepoAZB/resources/webhealth_lambda.py:19]: I'll do that. Much thanks!

**Pull request merged**